.github/workflows/test.yml

```diff
@@ -2,29 +2,34 @@ name: Tests on push
 on:
   push:
     branches-ignore:    
-      - 'release/*' 
+      - 'release/*'
+  pull_request:
```

Comment:
this configuration add trigger to run test on pull request open, to avoid forgot fix test before merge a PR